app/helpers/application_helper.rb

```diff
@@ -201,4 +201,12 @@ def filters_params
     end
     filters_params
   end
+
+  def short_uuid_with_title(uuid)
+    content_tag(:abbr, format_uuid(uuid), title: uuid)
+  end
+  
+  def format_uuid(uuid)
+    uuid.nil? ? "" : "#{uuid[0,4]}…#{uuid[-4,4]}"
```

Comment:
I believe the nil check should be moved to `short_uuid_with_title`. If the UUID is missing, there shouldn't be any output at all. The way it is now, a missing UUID would produce `<abbr title=""></abbr>` which is no good.